### PR TITLE
Covid 19 info

### PR DESCRIPTION
**Pull request by Mucyosoda**
**Covid19 app**  provides all information numerically in each country, the first-page displays all the countries, when you click on one country the other page with country info will display. the page had a search option where you can search by country name.
Countries and details are gotten and stored in the API.
App built in:
- React
- Redux
- JavaScript
- CSS
- Linter
- Baberlc
- API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,106 @@
-# covid-19-world-wide
-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (Meltrust) [inline README.md:1]: - [ ] Kindly please make a video for this project., according to and following closely these requirements:


image

- [ ] Let's give credit to this design's author! According to these instructions:


image


- [ ] You need to please improve your readme.   Please kindly follow this [template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

That way, your app will be well documented 👍

File: src/CSS/countryzz.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  padding-block: 1rem;
+}
+
+.world-cases {
+  font-weight: bolder;
+  color: #000;
+  font-family: 'lato', sans-serif;
+  padding: 4px;
+}
+
```
Comment (Meltrust) [inline src/CSS/countryzz.css]: - [ ] Please, kindly correct this part on your homepage:


image

As you can see, the elements are not centered, some of them contain an overflow.  
If you do this, your homepage will look striking! 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Mucyosoda, 


## Almost there! Keep it up!!!

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AshakaE** [comment]
### Hi @Mucyosoda   🙋‍♀️, 

# STATUS APPROVED ✔

Your:
   - Project is impressive 👏
   - Git flow ✔️ 
   - Pull request 👍 
   - README 👌 
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**